### DISCUSSION TITLE
Added wife to firefly but it's asking for bank info again

### DISCUSSION THREAD

**Discussion by Dylon0107** [firefly-iii/firefly-iii#9493] [Firefly III] [answered]
I had my wife make an account on our Firefly so she can sign in on her own to all of our budgeting information. After I made her account it's trying to get her to re-add all of our bank info again. 

How do I fix this so that it just shows her our budget instead of trying to get her to recreate a new one?

**Comment by JC5** [comment]
Hi, thanks for opening a discussion. 

https://docs.firefly-iii.org/how-to/firefly-iii/features/multi-user/

**Reply by Dylon0107** [reply]
It's still acting for her as its own instance with her own information. None of the information I've added is there nor the banks I've added

**Reply by JC5** [reply, marked as answer]
Yes that's what it says on that page.

**Reply by sebadamus** [reply]
Hi, I have just installed Firefly and intended to use it like @Dylon0107, meaning that my fiancee and I wanted to share the same instance/budget, but the software is not just like that I understood.

I think the only way is to share your user login, so everyone that logins with the same login will have the same data to work on.

Maybe it would be a future feature? (or I miss understood)

PS: will start helping with the spanish translation :-)

**Reply by rkoosaar** [reply]
I have also been thinking about a similar issue. I have setup my accounts via the 'owner' account. It would be really nice if you could setup another user that has either read-write or read-only access to those accounts (its not exactly the same multi user setup you have listed in docs) (this might need a Feature request I guess

**Reply by JC5** [reply]
Like it says on the page, this is something I'm working on :)
